simphony_mayavi/cuds/vtk_lattice.py

```diff
@@ -53,29 +64,35 @@ def __init__(self, name, type_, data_set, data=None):
         self.point_data = CubaData(
             data, stored_cuba=self.supported_cuba, size=size)
 
-        # Estimate the lattice parameters
-        if self.type in ('Cubic', 'OrthorombicP', 'Square', 'Rectangular'):
+        # Determine origin
+        try:
+            self._origin = self.data_set.origin
+        except AttributeError:
```

Comment:
You're right.  I should instead insert these two lines in the next block when the type of data_set is determined.